Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -3395,6 +3406,11 @@ bool AccessibilityObject::isExpanded() const
     }
 
     if (supportsExpanded()) {
+        if (RefPtr commandForElement = this->commandForElement()) {
+            if (RefPtr dialogElement = dynamicDowncast<HTMLDialogElement>(commandForElement); dialogElement && commandType() == CommandType::Close)
```

Comment:
If #41288 gets merged first this will need to account for that too (and also the test will need updating)

Reply:
No longer applicable